test/Yandex/Allure/Adapter/Annotation/AnnotationProviderTest.php

```diff
@@ -41,16 +42,14 @@ public function testGetMethodAnnotations()
         $this->assertEquals(self::TYPE_METHOD, $annotation->value);
     }
 
-    /**
-     * @expectedException \Doctrine\Common\Annotations\AnnotationException
-     */
-    public function testShouldThrowExceptionForNotImportedAnnotations()
+    public function testShouldThrowExceptionForNotImportedAnnotations(): void
     {
         $instance = new Fixtures\ClassWithIgnoreAnnotation();
+        $this->expectException(AnnotationException::class);
```

Comment:
Using annotations to intercept exceptions is deprecated in PHPUnit 8.0 and is removed in 9.0.